.gitignore

```diff
@@ -1,9 +1,58 @@
-.DS_Store
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+
+# exposed public scripts except sw.js
+/public/scripts/*
+!/public/scripts/sw.js
+
+# env
+.env/.env.dev.local
+
+# testing
+/coverage
 
-node_modules
+# build
+/build
+/build-development
+/build-demo
+/build-staging
+/build-mobile-staging
+/build-ci
```

Comment:
so for this .gitignore, i thinkk most of this stuff got moved to the packages/web and packages/mobile sections. mind double checking on that?